src/App.vue

```diff
@@ -44,16 +42,15 @@ export default defineComponent({
         }
     },
     mounted() {
-        const chainSettings = useChainStore().currentChain.settings as EVMChainSettings;
-        // if the organization using this application is Telos, import Fathom analytics
-        if (TELOS_CHAIN_IDS.includes(chainSettings.getChainId())) {
-            const script = document.createElement('script');
-            script.src = 'https://cdn.usefathom.com/script.js';
-            script.dataset.site = 'ISPYEAKT';
-            script.dataset.spa = 'auto';
-            script.defer = true;
-            document.body.appendChild(script);
-        }
+
+        // Telos Wallet allows users to switch between Telos Zero and Telos EVM.
+        // The loading of this script should be independent of the chain the user is on.
+        const script = document.createElement('script');
+        script.src = 'https://cdn.usefathom.com/script.js';
+        script.dataset.site = 'ISPYEAKT';
+        script.dataset.spa = 'auto';
+        script.defer = true;
+        document.body.appendChild(script);
```

Comment:
@ezra-sg, this code was inside an if and the condition was that the chain used is EVM. 

I had a lot of problems related to the chain being set (especially on auto-login situations where telos-evm is set because of the UI and then the telos-zero is set because the user was previously logged in telos zero).

What I need to know is if it is safe to remove that condition and have this code executed always.